packages/core-utils/src/time.js

```diff
@@ -1,10 +1,60 @@
-import moment from "moment-timezone";
+import {
+  startOfDay,
+  add,
+  format,
+  formatDuration as dateFnsFormatDuration
+} from "date-fns";
+import { utcToZonedTime } from "date-fns-tz";
 
 // special constants for making sure the following date format is always sent to
 // OTP regardless of whatever the user has configured as the display format
 export const OTP_API_DATE_FORMAT = "YYYY-MM-DD";
+// Date-Fns uses a different string format that moment.js
+// see https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md
+export const OTP_API_DATE_FORMAT_DATE_FNS = "yyyy-MM-dd";
 export const OTP_API_TIME_FORMAT = "HH:mm";
 
+/**
+ * To ease the transition away from moment.js, this method uses date-fns to format durations
+ * the way moment.js did.
+ * @param {number}  seconds     The number of seconds to format
+ * @param {boolean} showSeconds Whether to render seconds or not
+ * @param {boolean} localize    If true, will create output like moment.js using date-fns locale.
+ * Otherwise, uses date-fns default
+ * @returns                   Formatted duration
+ */
+function formatDurationLikeMoment(seconds, showSeconds, localize = true) {
+  const hours = Math.floor(seconds / 3600);
+  const minutes = Math.floor((seconds - hours * 3600) / 60);
+  const secondsLeftOver = showSeconds
+    ? seconds - hours * 3600 - minutes * 60
+    : 0;
+  const specLookup = {
+    xHours: "hr",
+    xMinutes: "min",
+    xSeconds: "sec"
+  };
+  const locale = localize
+    ? {
+        code: "en-US",
+        formatDistance: (spec, val) => {
+          return `${val} ${specLookup[spec]}`;
+        }
+      }
+    : undefined;
```

Comment:
How will this change (if at all) once localization is in place across otp-ui? Might be worth coordinating with @binh-dam-ibigroup on this if you haven't already to future proof this approach.

Reply:
We have not touched distance stuff in otp-react-redux yet, but we have [implemented this for formatting durations](https://github.com/opentripplanner/otp-react-redux/blob/dev/lib/components/narrative/default/default-itinerary.js#L51). And there will be a tipping point where we will have to merge that with core-utils.